Suggested edits (#39)
apeltzer/eager2-paper: content/02.introduction.md

streamlined and efficient pipelines can help increase accessibility to
high-quality analyses.

The degraded nature of aDNA poses an extra layer of complexity to standard
modern genomic analysis. Through a variety of processes [@doi:10.1038/362709a0],
DNA molecules fragment over time, resulting in ultra-short molecules
[@doi:10.1038/nature17405]. These sequences have low nucleotide complexity
making it difficult to identify with precision which part of the genome a read
is derived from. Furthermore, when this fragmentation is not a 'clean break' it can
lead to uneven ends with single-stranded 'overhangs' at end of DNA molecules. These 
overhangs are susceptible to chemical processes, such as deamination, that lead 
to mis-incorporation of bases during genomic library construction
[@doi:10.1073/pnas.0704665104]. On top of this, taphonomic processes such as
heat, moisture, and microbial- and burial-environment processes lead to varying
rates of degradation [@doi:10.1093/nar/gkx361;
@doi:10.1146/annurev-genom-091416-035526], where the original DNA content of a
sample is lost and supplanted by present-day environmental DNA. Later handling by

> archaeologists, museum curators, and specialists responsible for the laboratory work can also contribute traces of 

Reviewer: ```suggestion
archaeologists, museum curators, and scientists can also contribute traces of 
```
Bioinformatician bias there ;). I still collected samples and handled them during sampling before sent to the lab.

Reviewer: If you don't like differentitation of archaeologists vs specialists, could say archaeogeneticists instead.

Author: I don't like the differentiation between archaeologists and scientists.
Archaeogeneticists would work

@doi:10.7554/eLife.36666]. Without easy access to the latest field-established
analytical routines, aDNA studies risk being published without the necessary
quality control checks that ensure aDNA authenticity and without yielding the

> full range of possibilities from their data. **[Suggestion: mention that given that material is limited, the second point is even more immportant.]**

Reviewer 2: ```suggestion
full range of possibilities from their data. A further point would be that given that material from samples is limited there are both ethical as well as economical interests to maximize yield [@doi:10.3390/genes8070180].
```

To address these shortcomings, we have completely re-implemented the latest
version of the EAGER pipeline in Nextflow [@doi:10.1038/nbt.3820] (a

> domain-specific-language (DSL)), specifically designed for the construction of

Reviewer: ```suggestion
domain-specific-language (DSL), specifically designed for the construction of
```

Close bracket comes later

@doi:10.7554/eLife.36666]. Without easy access to the latest field-established
analytical routines, aDNA studies risk being published without the necessary
quality control checks that ensure aDNA authenticity and without yielding the

> full range of possibilities from their data. A further point would be that given that material from samples is limited there are both ethical as well as economical interests to maximize yield [@doi:10.3390/genes8070180].

Reviewer: ```suggestion
full range of possibilities from their data. A further point would be that given that material from samples is limited, there are both ethical as well as economical interests to maximize yield [@doi:10.3390/genes8070180].
```
